modules/internal/ChapelRange.chpl

```diff
@@ -1722,6 +1705,76 @@ operator :(r: range(?), type t: range(?)) throws where !chpl_castIsSafe(r, t) {
   return tmp;
 }
 
+/* Cast a range to a new range type.  The overload below throws when
+   the original bounds and/or stride do not fit in the new type or 'strides'.
+   TODO: should we allow 't' to be generic?
+ */
```

Comment:
This has a doc comment but is marked nodoc?

Reply:
It was this way a long time ago. My understanding is that this comment is aimed at the developers of ChapelRange, not users of ranges.